test/system/040-serve.bats

```diff
@@ -121,6 +121,9 @@ verify_begin=".*run --rm"
     run_ramalama ps
     is "$output" ".*${container1}" "list correct for container1"
 
+    run_ramalama chat --ls
+    is "$output" "ollama://smollm:135m" "list of models available correct"
+
```

Comment:
While you are in, could you add test for

run_ramalama run --model bogus

And verify an error message.

Reply:
This was the format prior to this PR (talking about chat only for a moment):

ramalama chat MODEL [ARGS]

It doesn't make sense that MODEL is mandatory when trying to list various models via:

ramalama chat --ls

Or when connecting to llama-server which already has a model loaded.

So I'm proposing making this breaking change to "ramalama chat" now, nobody really knows "ramalama chat" exists yet.

"ramalama run" is a little different because it's serving up a model.